custom_components/frigate/__init__.py

```diff
@@ -172,11 +172,11 @@ async def async_setup(hass: HomeAssistant, config: Config) -> bool:
     ws_api_async_setup(hass)
 
     session = async_get_clientsession(hass)
-    hass.http.register_view(JSMPEGProxyView(session))
-    hass.http.register_view(NotificationsProxyView(session))
-    hass.http.register_view(SnapshotsProxyView(session))
-    hass.http.register_view(VodProxyView(session))
-    hass.http.register_view(VodSegmentProxyView(session))
+    hass.http.register_view(JSMPEGProxyView(session, chunk_response=True))
```

Comment:
What is motivating the change to support chunk and non-chunk (and is it worth the extra complexity)?

Reply:
I think it's more efficient to just leave the chunking to the lower layer. Otherwise it's looping in python, which is not super efficient...4mb of data would be 1000 times through that loop, with the potential overhead of giving up async context each time.
I'm assuming the main point of using a streaming response is for the video assets.
But you're right, it might not be worth the extra complexity. An alternative would be to just set the chunk size a bit bigger, maybe 40-100kb or so?

Reply:
Actually, I just pushed a commit that reverts that complexity and just uses StreamResponse, but it now uses iter_any() which I think makes the most sense.

Reply:
I'll pull the iter_any() change out and submit it as a separate PR as it can be applied now, independently.